File: exams/frmMeritListConfig.vb
Public Class frmMeritListConfig

    Private Sub frmMeritListConfig_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        If Not connect() Then
            Me.Close()
        Else
            load_class(ComboBox1)
        End If
    End Sub
    Private Sub get_merit_list_configuration()
        qread("SELECT * FROM merit_list_config WHERE class='" & escape_string(ComboBox1.SelectedItem) & "'")
        If dbreader.RecordsAffected > 0 Then
            dbreader.Read()
            chkSE.Checked = dbreader("se")
            chkTP.Checked = dbreader("tp")
            chkMP.Checked = dbreader("mp")
            chkMM.Checked = dbreader("mm")
            chkMG.Checked = dbreader("mg")
            chkTM.Checked = dbreader("tm")
            chkStr.Checked = dbreader("str")
            chkSP.Checked = dbreader("sp")
            chkOP.Checked = dbreader("op")
            chkVAP.Checked = dbreader("vap")
            chkKCPE.Checked = dbreader("kcpe")
            chkIndex.Checked = dbreader("index_no")
        Else
            chkSE.Checked = False
            chkTP.Checked = False
            chkMP.Checked = False
            chkMM.Checked = False
            chkMG.Checked = False
            chkTM.Checked = False
            chkStr.Checked = False
            chkSP.Checked = False
            chkOP.Checked = False
            chkVAP.Checked = False
            chkKCPE.Checked = False
            chkIndex.Checked = False
        End If
    End Sub
    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        If qwrite("DELETE FROM merit_list_config WHERE `class` = '" & escape_string(ComboBox1.SelectedItem) & "';") And qwrite("INSERT INTO `merit_list_config` (`class`, `se`, `tp`, `mp`, `mm`, `mg`, `tm`, `str`, `sp`, `op`, `vap`, `kcpe`, `index_no`)" &
                                                                   "VALUES ('" & escape_string(ComboBox1.SelectedItem) & "', '" & chkSE.Checked & "', '" & chkTP.Checked & "', '" & chkMP.Checked & "', '" & chkMM.Checked &
                                                                   "', '" & chkMG.Checked & "', '" & chkTM.Checked & "', '" & chkStr.Checked & "', '" & chkSP.Checked &
                                                                   "', '" & chkOP.Checked & "', '" & chkVAP.Checked & "', '" & chkKCPE.Checked & "','" & chkIndex.Checked & "');") Then
            success("Configuration Details Successfully Saved!")
        Else
            failure("Configuration Details Could Not Be Saved!")
        End If
    End Sub

    Private Sub ComboBox1_SelectedIndexChanged(sender As Object, e As EventArgs) Handles ComboBox1.SelectedIndexChanged
        get_merit_list_configuration()
    End Sub
End Class

File: exams/frmMeritListConfig.cs
using System;
using Microsoft.VisualBasic.CompilerServices;

namespace exams
{
    public partial class frmMeritListConfig
    {
        public frmMeritListConfig()
        {
            InitializeComponent();
            _ComboBox1.Name = "ComboBox1";
            _Button1.Name = "Button1";
        }

        private void frmMeritListConfig_Load(object sender, EventArgs e)
        {
            if (!publicSubsNFunctions.connect())
            {
                Close();
            }
            else
            {
                var argcbo = ComboBox1;
                publicSubsNFunctions.load_class(ref argcbo);
                ComboBox1 = argcbo;
            }
        }

        private void get_merit_list_configuration()
        {
            string argq = "SELECT * FROM merit_list_config WHERE class='" + publicSubsNFunctions.escape_string(Conversions.ToString(ComboBox1.SelectedItem)) + "'";
            publicSubsNFunctions.qread(ref argq);
            if (publicSubsNFunctions.dbreader.RecordsAffected > 0)
            {
                publicSubsNFunctions.dbreader.Read();
                chkSE.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["se"]);
                chkTP.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["tp"]);
                chkMP.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["mp"]);
                chkMM.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["mm"]);
                chkMG.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["mg"]);
                chkTM.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["tm"]);
                chkStr.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["str"]);
                chkSP.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["sp"]);
                chkOP.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["op"]);
                chkVAP.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["vap"]);
                chkKCPE.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["kcpe"]);
                chkIndex.Checked = Conversions.ToBoolean(publicSubsNFunctions.dbreader["index_no"]);
            }
            else
            {
                chkSE.Checked = false;
                chkTP.Checked = false;
                chkMP.Checked = false;
                chkMM.Checked = false;
                chkMG.Checked = false;
                chkTM.Checked = false;
                chkStr.Checked = false;
                chkSP.Checked = false;
                chkOP.Checked = false;
                chkVAP.Checked = false;
                chkKCPE.Checked = false;
                chkIndex.Checked = false;
            }
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            if (publicSubsNFunctions.qwrite("DELETE FROM merit_list_config WHERE `class` = '" + publicSubsNFunctions.escape_string(Conversions.ToString(ComboBox1.SelectedItem)) + "';") & publicSubsNFunctions.qwrite("INSERT INTO `merit_list_config` (`class`, `se`, `tp`, `mp`, `mm`, `mg`, `tm`, `str`, `sp`, `op`, `vap`, `kcpe`, `index_no`)" + "VALUES ('" + publicSubsNFunctions.escape_string(Conversions.ToString(ComboBox1.SelectedItem)) + "', '" + chkSE.Checked + "', '" + chkTP.Checked + "', '" + chkMP.Checked + "', '" + chkMM.Checked + "', '" + chkMG.Checked + "', '" + chkTM.Checked + "', '" + chkStr.Checked + "', '" + chkSP.Checked + "', '" + chkOP.Checked + "', '" + chkVAP.Checked + "', '" + chkKCPE.Checked + "','" + chkIndex.Checked + "');"))
            {
                publicSubsNFunctions.success("Configuration Details Successfully Saved!");
            }
            else
            {
                publicSubsNFunctions.failure("Configuration Details Could Not Be Saved!");
            }
        }

        private void ComboBox1_SelectedIndexChanged(object sender, EventArgs e)
        {
            get_merit_list_configuration();
        }
    }
}